core/consensus/src/engine.rs

```diff
@@ -585,8 +637,40 @@ impl<Adapter: ConsensusAdapter + 'static> ConsensusEngine<Adapter> {
         );
 
         let block_hash = Hash::digest(block.encode_fixed()?);
+
+        log::info!(
+            "[consensus] update_status for handle_commit, height {}, block:{:?}, proof:{:?}",
+            block.header.height,
+            block.clone(),
+            proof.clone()
+        );
+
         self.status_agent
-            .update_by_commited(metadata.clone(), block, block_hash, proof);
+            .update_by_committed(metadata.clone(), block, block_hash, proof);
+
+        log::info!(
+            "[consensus] committed, consented height {}, proof height {}, status_agent:{:?}",
+            self.status_agent.to_inner().current_height,
```

Comment:
Avoid unnecessary `to_inner`.

Reply:
fixed, to avoid use read() which will impact performance